train_notebook.ipynb

```diff
@@ -0,0 +1,488 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'start training with parameters : {Parameters.as_dict()}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_train, y_train, cols_train = train_data_loader(\n",
+    "    DATALAKE_CHANNEL_ID, DATALAKE_TRAIN_FILE_ID, LABEL_FIELD, INPUT_FIELDS)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "X_train.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "y_train"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dtrain = lgb.Dataset(X_train, y_train)\n",
+    "\n",
+    "if DATALAKE_VAL_FILE_ID:\n",
+    "    X_val, y_val, _ = train_data_loader(\n",
+    "        DATALAKE_CHANNEL_ID, DATALAKE_VAL_FILE_ID, LABEL_FIELD, INPUT_FIELDS)\n",
+    "else:\n",
+    "    X_val, y_val = None, None\n",
+    "\n",
+    "extraction_cb = ModelExtractionCallback()\n",
+    "tensorboard_cb = TensorBoardCallback(statistics, writer)\n",
+    "tensorboard_cb.set_valid(X_val, y_val, Parameters.IS_CLASSIFICATION, IS_MULTI, Parameters.NUM_CLASS)\n",
+    "callbacks = [extraction_cb, tensorboard_cb,]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lgb.cv(PARAMS, dtrain, nfold=Parameters.NFOLD,\n",
+    "       early_stopping_rounds=Parameters.EARLY_STOPPING_ROUNDS,\n",
+    "       verbose_eval=Parameters.VERBOSE_EVAL,\n",
+    "       stratified=STRATIFIED,\n",
+    "       callbacks=callbacks,\n",
+    "       metrics=Parameters.METRIC,\n",
+    "       seed=Parameters.SEED)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "models = extraction_cb.raw_boosters\n",
+    "for i,model in enumerate(models):\n",
+    "    model.save_model(os.path.join(ABEJA_TRAINING_RESULT_DIR, f'model_{i}.txt'))\n",
+    "\n",
+    "di = {\n",
+    "    **(Parameters.as_dict()),\n",
+    "    'cols_train': cols_train\n",
+    "}\n",
+    "lgb_env = open(os.path.join(ABEJA_TRAINING_RESULT_DIR, 'lgb_env.json'), 'w')\n",
+    "json.dump(di, lgb_env)\n",
+    "lgb_env.close()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def handler(context):\n",
+    "    print(\"finish.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "source": [
+    "## Step 2: Run this on ABEJA Platform\n",
```

Comment:
Step 2 is the key component of this PR.
I added the tag of `remove_cell` not to include unnecessary cells.